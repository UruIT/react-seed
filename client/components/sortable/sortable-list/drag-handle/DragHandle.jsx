import React from 'react';
import { SortableHandle } from 'react-sortable-hoc';

import styles from './drag-handle.scss';

const DragHandle = SortableHandle(() => <i className={styles.dragHandle} />);

export default DragHandle;
