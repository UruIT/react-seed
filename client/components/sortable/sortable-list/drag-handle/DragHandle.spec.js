import React from 'react';
import renderer from 'react-test-renderer';

import DragHandle from './DragHandle';

describe('<DragHandle />', () => {
	it('snapshot', () => {
		const tree = renderer.create(<DragHandle />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
