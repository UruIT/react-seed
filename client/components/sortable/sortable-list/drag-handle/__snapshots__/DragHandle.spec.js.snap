// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DragHandle /> snapshot 1`] = `
<i
  className="dragHandle"
/>
`;
